webdriver/tests/get_element_shadow_root/get.py

```diff
@@ -19,7 +19,7 @@ def test_no_top_browsing_context(session, closed_window):
 
     session.window_handle = original_handle
     response = get_shadow_root(session, element.id)
-    assert_error(response, "no such element")
+    assert_error(response, "stale element reference")
```

Comment:
The error type as used before was correct given that the window handle has been changed in line 20. Only for the same handle the `stale element` would be raised.